mlos_core/mlos_core/util.py

```diff
@@ -10,9 +10,9 @@
 from ConfigSpace import Configuration, ConfigurationSpace
 
 
-def config_to_dataframe(config: Configuration) -> pd.DataFrame:
+def config_to_series(config: Configuration) -> pd.Series:
```

Comment:
This gets used in the internal representation and in some of the test cases. The main reason to have it out is because of the test cases.

Reply:
static methods can be called in test methods too.

It's purely an organizational thing.

Reply:
I think that this function fits with all of the other functions in this file. It is used across multiple classes, and it is used in multiple test cases, but it would not be used outside of mlos_core

Reply:
Let's just leave it as is for now for now so we can get this in.  Thanks!